managing-os/templates/archlinux.md

```diff
@@ -16,27 +16,647 @@ be considered experimental as Qubes developers team use mainly Fedora-based VMs
+    the Fedora 23 template. I named the VM “**development**”. These instructions assume a standalone VM based on a       Fedora template is being used.
+<br>
+<br>
+<a href="http://i256.photobucket.com/albums/hh165/tim_welter/Qubes%20OS/createstandalonevm_zpscf78sezy.png" target="_blank"><img src="http://i256.photobucket.com/albums/hh165/tim_welter/Qubes%20OS/createstandalonevm_zpscf78sezy.png"/></a>
+<br>
+<br>
+*   Ensure there is at least 25GB preferably 30GB of free space in the private storage. I made the private storage 30GB to be safe.
+<br>
+<br>
+<a href="http://i256.photobucket.com/albums/hh165/tim_welter/Qubes%20OS/development30gb_zpsijgeanf4.png" target="_blank"><img src="http://i256.photobucket.com/albums/hh165/tim_welter/Qubes%20OS/development30gb_zpsijgeanf4.png"/></a>
+<br>
+<br> 
+
+*Note: Unless otherwise noted,  all commands are from within the “development” VM or whatever you named your standalone VM used for building the template.*
+<br>
+<br>
+<br>
+
+<big>**2:   Create GitHub Account(optional):**</big>
+
+*   It can be helpful. Creating only a basic account is all that is needed. This will allow you to help, going           forward, with the Qubes project. You could be help edit errors in documentation. It can also be of use building      other templates.
+    
+*   Create user account here https://github.com
+<br>
+<br>
+<a href="http://i256.photobucket.com/albums/hh165/tim_welter/Qubes%20OS/joingithub_zpsowemb6wb.png" target="_blank"><img src="http://i256.photobucket.com/albums/hh165/tim_welter/Qubes%20OS/joingithub_zpsowemb6wb.png"/></a>    
+<br>
+<br>
+<br>
+
+<big>**3:   Install necessary packages in the 'development' VM to use "Qubes Automated Build System":**</big>
+
+*   Necessary packages to install:
+
+    *   git
+    
+    *   createrepo
+    
+    *   rpm-build
+    
+    *   make
+    
+    *   rpmdevtools
+
+    *   python-sh
+
+    *   dailog
+
+    *   rpm-sign
+<br>
+
+
+*   The tools can usually be installed all together with the following terminal command string:
+
+    *   **$ sudo dnf install git createrepo rpm-build make wget rpmdevtools python-sh dialog rpm-sign**
+<br>
+<br>
+<a href="http://i256.photobucket.com/albums/hh165/tim_welter/Qubes%20OS/compiletools_zps5xojzr6w.png" target="_blank"><img src="http://i256.photobucket.com/albums/hh165/tim_welter/Qubes%20OS/compiletools_zps5xojzr6w.png"/></a>
+<br>
+<br>
+<br>
+
+<big>**4:   Installing the "Qubes Automated Build System":**</big>
+
+*   To get the most current build system its best to use marmarek's git repository.
+
+    *   $ **git clone https://github.com/marmarek/qubes-builder.git**
+<br>
+<br>
+<a href="http://i256.photobucket.com/albums/hh165/tim_welter/Qubes%20OS/gitclonemarmarek_zpsrrbkfnjp.png" target="_blank"><img src="http://i256.photobucket.com/albums/hh165/tim_welter/Qubes%20OS/gitclonemarmarek_zpsrrbkfnjp.png"/></a>
+<br>
+<br>
+
```

Comment:
Here should signature verification of cloned repository, see: https://www.qubes-os.org/doc/qubes-builder/